src/App.tsx

```diff
@@ -57,6 +60,8 @@ export enum Routes {
   EVENT_REGISTRATIONS = '/events/:id/rsvp',
   FAMILY_DETAILS = '/family-details/:id',
   CREATE_EVENT = '/create-event',
+  EDIT_EVENT = '/events/:id/edit',
```

Comment:
🔧 Can we make this `/edit-event/:id` instead?

Reply:
Should we change EVENT_REGISTRATIONS (line 60) to match? We followed that convention when creating this one.